testing/tests/DevExpress.viz.sparklines/bulletTooltip.tests.js

```diff
@@ -47,77 +80,91 @@ const environment = {
         value: 10,
         maxScaleValue: 20,
         tooltip: {
             font: {
                 size: 12
-            },
-            enabled: true
+            }
         }
     });
-    bullet._showTooltip();
+    showBulletTooltip(bullet);
+    const tooltip = getBulletTooltip(bullet);
 
-    const arg = bullet._tooltip.ctorArgs;
+    const arg = tooltip.ctorArgs;
     assert.strictEqual(arg.length, 1);
     assert.deepEqual(arg[0].cssClass, 'dxb-tooltip', 'parameter - cssClass');
     assert.strictEqual(arg[0].eventTrigger, bullet._eventTrigger, 'parameter - event trigger');
+});
 
-    assert.equal(bullet._tooltip.update.callCount, 1, 'update is called');
-    assert.ok($.isFunction(bullet._tooltip.update.lastCall.args[0].customizeTooltip));
-    bullet._tooltip.update.lastCall.args[0].customizeTooltip = {};
-    assert.deepEqual(bullet._tooltip.update.lastCall.args[0], {
-        enabled: true,
-        font: {
-            size: 12
-        },
-        customizeTooltip: {}
+QUnit.test('Update method shoul accept valid params when tooltip enabled', function(assert) {
+    const bullet = this.createBullet({
+        value: 10,
+        maxScaleValue: 20,
+        tooltip: {
+            font: {
+                size: 12
+            }
+        }
     });
+    showBulletTooltip(bullet);
+    const tooltip = getBulletTooltip(bullet);
+
+    assert.equal(tooltip.update.callCount, 1, 'update is called');
+    assert.strictEqual(tooltip.update.lastCall.args[0].enabled, true);
+    assert.strictEqual(tooltip.update.lastCall.args[0].font.size, 12);
+    assert.strictEqual(isFunction(tooltip.update.lastCall.args[0].contentTemplate), true);
 });
 
-QUnit.test('Enabled tooltip. Empty data', function(assert) {
+QUnit.test('Tooltip constructor should accept valid parems when tooltip enabled and no data', function(assert) {
     const bullet = this.createBullet({
         tooltip: {
             font: {
                 size: 12
-            },
-            enabled: true
+            }
         }
     });
-    bullet._showTooltip();
+    showBulletTooltip(bullet);
+    const tooltip = getBulletTooltip(bullet);
 
-    const arg = bullet._tooltip.ctorArgs;
+    const arg = tooltip.ctorArgs;
     assert.strictEqual(arg.length, 1);
     assert.deepEqual(arg[0].cssClass, 'dxb-tooltip', 'parameter - cssClass');
     assert.strictEqual(arg[0].eventTrigger, bullet._eventTrigger, 'parameter - event trigger');
+});
 
-    assert.equal(bullet._tooltip.update.callCount, 1, 'update is called');
-    assert.ok($.isFunction(bullet._tooltip.update.lastCall.args[0].customizeTooltip));
-    bullet._tooltip.update.lastCall.args[0].customizeTooltip = {};
-    assert.deepEqual(bullet._tooltip.update.lastCall.args[0], {
-        enabled: false,
-        font: {
-            size: 12
-        },
-        customizeTooltip: {}
+QUnit.test('Update method shoul accept valid params when tooltip enabled and no data', function(assert) {
```

Comment:
plz, check everywhere
```suggestion
QUnit.test('Update method should accept valid params when tooltip is enabled and no data', function(assert) {
```